auctions/models.py

```diff
@@ -567,52 +552,43 @@ def request(self):
         codesy_fee = PayoutFee(
             payout=self,
             fee_type='codesy',
-            amount=round_penny(total_payout_amount * Decimal('0.025')),
+            amount=round_penny(total_payout_amount * codesy_pct),
         )
         codesy_fee.save()
 
-        total_fees = paypal_fee.amount + codesy_fee.amount
-        self.charge_amount = total_payout_amount - total_fees
-        self.save()
+        stripe_fee_amount = round_penny(
+            (total_payout_amount * stripe_pct) + stripe_transaction
+        )
+
+        stripe_fee = PayoutFee(
+            payout=self,
+            fee_type='Stripe',
+            amount=stripe_fee_amount
+        )
+        stripe_fee.save()
 
-        # TEMPORARY WHILE WAITING FOR PAYPAL
-        # return False
-        # attempt paypal payout
-        # user generated id sent to paypal is limited to 30 chars
-        sender_id = self.short_key()
-        paypal_payout = PaypalPayout({
-            "sender_batch_header": {
-                "sender_batch_id": sender_id,
-                "email_subject": "Your codesy payout is here!"
-            },
-            "items": [
-                {
-                    "recipient_type": "EMAIL",
-                    "amount": {
-                        "value": int(self.charge_amount),
-                        "currency": "USD"
-                    },
-                    "receiver": receiver,
-                    "note": "Here's your payout for fixing an issue.",
-                    "sender_item_id": sender_id
-                }
-            ]
-        })
         try:
-            payout_attempt = paypal_payout.create(sync_mode=True)
-        except:
-            payout_attempt = False
-
-        if payout_attempt:
-            if paypal_payout.items:
-                for item in paypal_payout.items:
-                    if item.transaction_status == "SUCCESS":
-                        self.api_success = True
-                        self.confirmation = item.payout_item_id
-                        self.save()
-                    else:
-                        payout_attempt = False
-        return payout_attempt
+            transfer = stripe.Transfer.create(
+                currency="usd",
+                description="codesy.io payout",
+                amount=int(total_payout_amount * 100),
+                application_fee=int(
+                    (codesy_fee.amount + stripe_fee.amount) * 100),
+                destination=managed_account.account_id
+            )
+            if transfer:
+                self.charge_amount = (
+                    total_payout_amount
+                    - (codesy_fee.amount + stripe_fee.amount)
+                )
```

Comment:
Can we pull this out of the `transfer` object instead of re-calculating it again?